jib-core/src/main/java/com/google/cloud/tools/jib/configuration/LayerConfiguration.java

```diff
@@ -80,17 +84,14 @@ public Builder addEntry(Path sourceFile, AbsoluteUnixPath pathInContainer) {
      *     sourceFile}
      * @param permissions the file permissions on the container. If null, then default permissions
      *     are used (644 for files, 755 for directories)
-     * @param lastModified the modification time of the file
      * @return this
      * @see Builder#addEntry(Path, AbsoluteUnixPath)
+     * @see FilePermissions#DEFAULT_FILE_PERMISSIONS
+     * @see FilePermissions#DEFAULT_FOLDER_PERMISSIONS
      */
     public Builder addEntry(
-        Path sourceFile,
-        AbsoluteUnixPath pathInContainer,
-        @Nullable FilePermissions permissions,
-        long lastModified) {
-      layerEntries.add(new LayerEntry(sourceFile, pathInContainer, permissions, lastModified));
-      return this;
+        Path sourceFile, AbsoluteUnixPath pathInContainer, FilePermissions permissions) {
+      return addEntry(sourceFile, pathInContainer, permissions, Instant.ofEpochSecond(1));
```

Comment:
I think `LayerEntry.DEFAULT_FILE_TIMESTAMP_PROVIDER.apply(sourceFile, pathInContainer)` is better than `Instant.ofEpochSecond(1)` in the long run.